src/Bridges/ApplicationDI/RoutingExtension.php

```diff
@@ -55,6 +52,13 @@ public function loadConfiguration()
 	public function beforeCompile()
 	{
 		$builder = $this->getContainerBuilder();
+		$router = $builder->getDefinition($this->prefix('router'));
```

Comment:
:( yes, you're right. it had reason in previous version but after the last changes, it can be moved back into loadConfiguration. I am gonna change it withing a few minutes.